.gitlab/common/test_infra_version.yml

```diff
@@ -4,4 +4,4 @@ variables:
   # and check the job creating the image to make sure you have the right SHA prefix
   TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
   # Make sure to update test-infra-definitions version in go.mod as well
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: e488596ad7f3
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 0700b1afb854
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
There is probably an automated PR that bumps it too, can you close it?

Reply:
Yes I will. Thanks for letting me know